Procfile

```diff
@@ -1 +1 @@
-web: mix phx.server
\ No newline at end of file
+web: MIX_ENV=prod mix ecto.migrate && mix assets.deploy && mix phx.server
```

Comment:
I'm not sure if we need to run `assets.deploy` in Procfile.
The elixir_buildbpack file run this command already:
https://github.com/dwyl/auth/blob/main/elixir_buildpack.config